ckanext/harvest/harvesters/ckanharvester.py

```diff
@@ -246,8 +236,10 @@ def fetch_stage(self,harvest_object):
                                         (url, e),harvest_object)
             return None
 
+        content = json.loads(content)['result']
```

Comment:
This looks fragile, we shouldn't assume it will be a CKAN API response (could be a 404, etc)